pkg/settings/setting.go

```diff
@@ -95,7 +95,7 @@ var (
 	IgnoreNodeName                    = NewSetting("ignore-node-name", "") // nodes to ignore when syncing v1.node to v3.node
 	NoDefaultAdmin                    = NewSetting("no-default-admin", "")
 	RestrictedDefaultAdmin            = NewSetting("restricted-default-admin", "false") // When bootstrapping the admin for the first time, give them the global role restricted-admin
-	EKSUpstreamRefreshCron            = NewSetting("eks-refresh-cron", "*/5 * * * *")
```

Comment:
Might want to add logic in the controller to delete the remaining setting if it's still around.

Reply:
+1

Reply:
`clusterupstreamrefresher` was taken out from this PR and submitted as separate https://github.com/rancher/rancher/pull/31674
Regarding cleaning up I did already research and it seems that CronJob is not Kubernetes resource. It seems that there is no need to delete anything else, but I would like to discuss it